src/routes/dashboard/fragment-wizard/import-json-modal.tsx

```diff
@@ -244,13 +243,14 @@ export const ImportJsonModal = (props: ImportJsonModalProps) => {
 									iconDescription={iconDescription}
 									invalid={invalid}
 									onDelete={handleFileUploaderItemClick}
+									{...rest}
```

Comment:
This looks more significant than just code style 🤔

Reply:
Changed order - so props are not over written, assuming rest contains keys like onDelete or size. Or might be safe to just remove since it wasn't being used 🤔